docs/docsite/rst/dev_guide/developing_collections_distributing.rst

```diff
@@ -0,0 +1,241 @@
+.. _distributing_collections:
+
+************************
+Distributing collections
```

Comment:
Is distributing the right term here? I thought it was publishing as the activity a collection owner does. Just wondering if someon would search 'how do I distribute my collection' vs 'how do I publish my collection' ? I also don't see anything in official AH documentation or marketing details that calls it a distribution server?

Reply:
Yeah, I debated about that. Current thinking is that "distributing" includes both creating the tarball and publishing the tarball to Galaxy/AH. I made sure the first paragraph includes both terms. Looking at it again, though, I see that I don't mention packaging it up to create the tarball . . . I'll add a sentence about that.

Reply:
oh, it's there, in the "three major steps" section . . . phew